languages/translations/vi.json

```diff
@@ -62,49 +62,57 @@
                 "default": "Đến nơi {nth}",
                 "upcoming": "Đến nơi {nth}",
                 "short": "Đến nơi",
-                "short-upcoming": "Đến nơi"
+                "short-upcoming": "Đến nơi",
+                "named": "Đến nơi {waypoint_name}"
```

Comment:
“Đến `{waypoint_name}`” would be better, but I can also fix this up in Transifex after the fact.